src/workers/atlas_worker.js

```diff
@@ -97,7 +97,8 @@ export default class AtlasWorker extends PeriodicWorker {
         return;
       }
       const challenges = await this.challengesRepository.ongoingChallenges();
-      await this.addLog(`Challenges preselected for resolution: ${challenges.length}`);
+      const recentlyFailedChallenges = challenges.filter(({challengeId}) => this.failedChallengesCache.failedChallengesEndTime[challengeId] !== undefined);
+      await this.addLog(`Challenges preselected for resolution: ${challenges.length} (out of ${recentlyFailedChallenges.length} failed recently)`);
```

Comment:
If I understand correctly, the message should be different:
```suggestion
      await this.addLog(`Challenges preselected for resolution: ${challenges.length} (out of which ${recentlyFailedChallenges.length} have failed recently)`);
```